scipy/stats/setup.py

```diff
@@ -96,6 +96,18 @@ def configuration(parent_package='', top_path=None):
     # Type stubs
     config.add_data_files('*.pyi')
 
+    # add levy stable module
+    config.add_data_files(join('_levy_stable', '*.py'))
```

Comment:
I think the reason this didn't work otherwise is that it's a separate submodule with an `__init__.py, and those are normally treated with `config.add_subpackage` (like right above in this `setup.py`).